Documentation/Signoffs/Bluetooth_Communication.md

```diff
@@ -0,0 +1,93 @@
+
+<sup>5</sup> Since the machine learning subsystem is ran on a personal laptop, the system must be in range to communicate. The laptop's location will be restricted to inside our target room.
+
+<sup>6</sup> In order to create a system that does not invade the privacy of the students and teachers in the room, the system will not store any sound data after it is used in processing. This will prevent the system from possibly storing private conversations that could be breached.
+
+<sup>7</sup> In order to create a system that does not invade the privacy of the students and teachers in the room, the system must encrypt any sound data sent wirelessly. This will help protect sound data from being easily broadcasted and received outside of the room.
+
+<sup>8</sup> The system has to communicate with the machine learning subsystem wirelessly in order to eliminate any need for wires or cables across the room. This removes physical obstructions in the walking area. This satisfies the accessibility ethics constraint.
+
+## Buildable schematic
+
+![image](/Documentation/Images/Bluetooth-Communication/Buildable-Schematics/BluetoothCommSchematic.png)  
+
+The above image is a schematic of the Arduino UNO R4 WiFi, the schematic focuses on the most important aspects of the device with respect to the overall design goal. The schematic also shows how the system connects to the main processor subsystem and power management subsystem. 
+
+## Analysis
+
+#### Power
+The system can be powered through a USB connection or through the onboard barrel jack connector[1] <sup>1</sup>. The recommended input voltage and current rating for the external power supply are 6-24 V. The subsystem will use the included AC/DC adapter as its power supply. Since the board needs to use the USB port in order to install the designed software, the subsystem will have to be programmed before being connected to the system's power supply to allow the subsystem to boot without being connected directly to a computer. 
+
+#### Bluetooth Capabilities
+The Arduino UNO R4 WiFi is able to transmit and receive in WiFi and Bluetooth 5 LE<sup>2</sup>. This is accomplished through the ESP32-S3-MINI[3]. Bluetooth 5.0 LE includes a PHY called LE 2M. This can support 2 Mb/s (2,000,000 bits per second). At the application layer, a data rate of up to approximately 1.4 Mb/s (1,400,000 bits per second) is possible[5]. Assuming the 2 second delay lines of 187.5 kiB [6], each send will be sending 187.5 kiB (or 1,536,000 bits). Therefore, Bluetooth 5 LE  will be able to send a full delay line in 1.10 seconds (1.536 Mb / 1.4 Mbps), and receive the filter index bits (also 187.5 kiB) in 1.10 seconds for a total of 2.20 seconds<sup>4</sup>.
```

Comment:
This is greater than the constraint. Should the constraint be changed? If so, explain why in the constraint section and make the change.